spec/admin/controllers/quizzes/index_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+require_relative '../../../../apps/admin/controllers/quizzes/index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.